tests/test_unit/test_auth/mocks/keycloak.py

```diff
@@ -0,0 +1,118 @@
+import json
+from base64 import b64encode
+
+import responses
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives.asymmetric import rsa
+from cryptography.hazmat.primitives.serialization import Encoding, PublicFormat
+from itsdangerous import TimestampSigner
+from jose import jwt
+
+# copied from .env.docker as backend tries to send requests to corresponding
+KEYCLOAK_CONFIG = {
+    "server_url": "http://keycloak:8080",
```

Comment:
If tests are run not inside docker container, `server_url` should be `http://localhost:8080/...` or something like that. Probably it's better to pass these options via TestSettings

Reply:
Now I see that this address is used only inside unit tests and mocked by `reponses`. So it actually can be hardcoded in a fixture.

Reply:
now get keycloak config only from `settings` fixture (`BackendSettings`) and mock requests to `keycloak:8080`